spec.html

```diff
@@ -7590,15 +7590,15 @@ <h1>GetThisEnvironment ( )</h1>
       <p>The abstract operation GetThisEnvironment takes no arguments. It finds the Environment Record that currently supplies the binding of the keyword `this`. It performs the following steps when called:</p>
       <emu-alg>
         1. Let _env_ be the running execution context's LexicalEnvironment.
-        1. Repeat,
+        1. [label="getthisenvironment-loop"] Repeat,
           1. Let _exists_ be _env_.HasThisBinding().
           1. If _exists_ is *true*, return _env_.
           1. Let _outer_ be _env_.[[OuterEnv]].
           1. Assert: _outer_ is not *null*.
           1. Set _env_ to _outer_.
       </emu-alg>
       <emu-note>
-        <p>The loop in step 2 will always terminate because the list of environments always ends with the global environment which has a `this` binding.</p>
+        <p>The loop in step <emu-xref href="#step-getthisenvironment-loop"></emu-xref> will always terminate because the list of environments always ends with the global environment which has a `this` binding.</p>
```

Comment:
```suggestion
        <p>The loop in step <emu-xref title href="#step-getthisenvironment-loop"></emu-xref> will always terminate because the list of environments always ends with the global environment which has a `this` binding.</p>
```